notebook/static/base/js/security.js

```diff
@@ -31,6 +31,10 @@ define([
                 }
             }
         }
+        // TODO(julienr): This is a ugly hack to work around the fact that
+        // by default caja doesn't seem to like base64 src for inline markdown
+        // images. Not sure if we should do that here
+        ATTRIBS['img::src'] = 0;
```

Comment:
Since this removes a security check, we need to make sure that there are no possible `image src:` attributes that can execute javascript before we can allow this.